main.rb

```diff
@@ -0,0 +1,23 @@
+def bubble_sort(arr)
+  number_ofloop = arr.length - 1
+  (0..number_ofloop).each do |_i|
+    (0...number_ofloop).each do |j|
+      arr[j], arr[j + 1] = arr[j + 1], arr[j] if arr[j] > arr[j + 1]
+    end
+  end
+  arr
+end
+
+puts bubble_sort([4, 3, 78, 2, 0, 2])
```

Comment:
- [ ] Please delete this line of debugging code (in the future you will learn to make automated tests for these cases). For now, if you want, you could save your examples in a different file and gitignore it (you will find it useful for enumerables project)
![image](https://user-images.githubusercontent.com/53256139/100308221-707b4680-2f6d-11eb-8c84-fbd63f02f03b.png)